runtime/memory.c

```diff
@@ -313,7 +295,7 @@ CAMLexport void caml_set_fields (value obj, value v)
   Assert (Is_block(obj));
 
```

Comment:
I think we should keep the mutable_stores stat collection:
```C
#if defined(COLLECT_STATS) && defined(NATIVE_CODE)
  Caml_state->mutable_stores++;
#endif
```
Cheap to maintain here and useful while `COLLECT_STATS` exists as an option.